src/components/screens/bookmarks/addBookmark/addBookmark.js

```diff
@@ -156,19 +158,22 @@ class AddBookmark extends React.Component {
   handleAddBookmark(e) {
     e.preventDefault();
     const {
-      token: { active }, bookmarkAdded, account, prevStep,
+      token: { active }, bookmarkAdded, account, prevStep, bookmarks,
     } = this.props;
     const { fields: { label, address } } = this.state;
     const { publicKey, delegate } = account.data;
-    bookmarkAdded({
-      token: active,
-      account: {
-        title: label.value,
-        address: address.value,
-        isDelegate: !!(delegate && delegate.username),
-        publicKey,
-      },
-    });
+
+    if (!bookmarks[active].some(bookmark => bookmark.address === address.value)) {
```

Comment:
@reyraa as I saw that it was possible to add multiple times the same address to bookmarks I added this code to prevent it from happening. I assume that it is not an expected feature, but let me know if it was intended to work that way.

Reply:
Thanks for detecting the issue. I think introducing a function to check if a bookmark exist is more readable than a conditional statement.